npm/svelte/rollup.config.mjs

```diff
@@ -1,3 +1,3 @@
 import { createEntries } from '@cypress/mount-utils/create-rollup-entry.mjs'
 
-export default createEntries({ formats: ['es', 'cjs'], input: 'src/index.ts' })
+export default createEntries({ formats: ['es', 'cjs'], input: 'src/index.ts', dtsOptions: { respectExternal: false } })
```

Comment:
can you link the git issue here as to why we needed to do this again?

Reply:
updated [60dd249](https://github.com/cypress-io/cypress/pull/30572/commits/60dd24993271dac64fbffc95702152a10adfb54b)